crates/mbe/src/mbe_expander/matcher.rs

```diff
@@ -356,6 +356,18 @@ impl<'a> TtIter<'a> {
             ExpandResult { value: _, err: Some(_) } => None,
         }
     }
+
+    pub(crate) fn eat_char(&mut self) -> Option<tt::TokenTree> {
```

Comment:
This should probably be named `eat_minus`, or take a `char` parameter that tells it which character to expect

Reply:
Ah... fixed.